wdfn-server/waterdata/tests/test_views.py

```diff
@@ -48,6 +49,13 @@ def test_everything_okay(self, r_mock):
         m_resp_param.iter_lines.return_value = iter(self.parameter_lines)
         r_mock.side_effect = [m_resp, m_resp_param]
 
+        m_resp_lookup = mock.Mock()
+        m_resp_lookup.status_code = 200
+        m_resp_lookup.return_value = {'Customers': [{'Name': 'mock_cooperator',
+                                                     'URL': 'http://www.fake.gov',
+                                                     'IconURL': 'http://www.fake.gov'}]}
+        r_mock_lookup.return_value.json = m_resp_lookup
```

Comment:
This doesn't make sense. . The return value for r_mock_lookup is just a dictionary not an Object where you would need to define the json function. I think you can eliminate lines 52, 53 and then combine 54 and 57 to assign r_mock_lookup.return_value to the dictionary.